src/gui/TrackContainerView.cpp

```diff
@@ -508,7 +508,22 @@ void TrackContainerView::scrollArea::wheelEvent( QWheelEvent * _we )
 	m_trackContainerView->wheelEvent( _we );
 	if( !_we->isAccepted() )
 	{
-		QScrollArea::wheelEvent( _we );
+		// Handle Shift+Wheel orientation of QWheelEvent
+		if (_we->modifiers() & Qt::ShiftModifier
+			&& _we->orientation() == Qt::Vertical)
+		{
+			// copy QWheelEvent but change orientation to Horizontal
+			QWheelEvent we(_we->posF(), _we->globalPosF(),
+				_we->pixelDelta(), _we->angleDelta(), _we->delta(),
+				Qt::Horizontal, _we->buttons(), _we->modifiers(),
+				_we->phase(), _we->source(), _we->inverted());
```

Comment:
It seems like `QWheelEvent::source()` was introduced in 5.5, and `inverted()` was in 5.7. It will make copying the wheel event complicated.

Reply:
Also, are you sure that you don't need to change the return values of `pixelDelta()`, `angleDelta()` and `delta()`?

Reply:
> It seems like `QWheelEvent::source()` was introduced in 5.5, and `inverted()` was in 5.7. It will make copying the wheel event complicated.

What version of Qt5 should we target? I'm using 5.12.4. I don't mind targeting a lower version.

> Also, are you sure that you don't need to change the return values of `pixelDelta()`, `angleDelta()` and `delta()`?

From all the testing I've done those seem to be okay and handled properly. As a note, `delta()` is obsolete in addition to the `orientation()`.

Reply:
> What version of Qt5 should we target?

The minimum is 5.6 LTS now, but we may consider bumping to 5.9 if necessary. For now, you may check for `QT_VERSION` to support Qt 5.6.

Reply:
Alright, I'll make sure I use a constructor from at most 5.6

Reply:
I think you should use that form only for 5.6 while keeping the code for 5.7 and upper.

Reply:
Ah, gotcha. Will do. 👍

Reply:
And you never did.